apps/vaporgui/PIntegerInputHLI.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#include "PWidgetHLI.h"
+#include "PIntegerInput.h"
+
+CreateHLI(PIntegerInput, long);
```

Comment:
I'm guessing there's a good reason for this, but I'm curious why each PWidget has its own file for these 4 lines of code?  Could this be consolidated in the relevant PWidget header file?   Or define individual instances of CreateHLI in a single header file?

Reply:
There are some minor benefits to doing this but mainly its just a design choice.

Reply:
Thanks.  Could you briefly elaborate on what the minor benefits are?  I'm fine resolving this, but I just don't know benefits.  I'm just concerned about the vaporgui directory getting unwieldy.

Reply:
One would be potentially improved compile time. Many vapor files import way more than they need to leading to increased compile times and more files that need to be recompiled when a header is modified.